newton/solvers/mujoco/solver_mujoco.py

```
@@ -1242,6 +1242,33 @@ def apply_mjc_control(model: Model, state: State, control: Control | None, mj_da
                 ],
                 device=model.device,
             )
+            
+            # Apply tendon control if available
+            if hasattr(control, 'tendon_target') and control.tendon_target is not None and model.tendon_actuator_count > 0:
+                # Use the tendon actuator mapping to set control values
+                tendon_targets = control.tendon_target.numpy()
+                tendon_mapping = model.mjc_tendon_actuator_to_actuator.numpy() if hasattr(model, 'mjc_tendon_actuator_to_actuator') else None
+                
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)